.github/workflows/ci-app-pr-environment-update.yml

```diff
@@ -0,0 +1,12 @@
+name: CI App PR Environment Update
+on:
+  workflow_dispatch:
+  # !! Uncomment the following lines once you've set up the dev environment and are ready to enable PR environments
+  # pull_request:
```

Comment:
Asking to make sure I understand: [The docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) say:

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. To trigger workflows by different activity types, use the types keyword.

But I didn't see a definition of what `synchronize` means. Is that basically mean when there is an update to the pull request that doesn't have a merge conflict?

Reply:
Yeah basically when you push changes to the PR